<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QToolBar" name="timeToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="speedDownDownAction"/>
   <addaction name="speedDownAction"/>
   <addaction name="separator"/>
   <addaction name="stopAction"/>
   <addaction name="startAction"/>
   <addaction name="separator"/>
   <addaction name="speedUpAction"/>
   <addaction name="speedUpUpAction"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="sceneToolBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="addItemAction"/>
   <addaction name="separator"/>
   <addaction name="saveAction"/>
   <addaction name="loadAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="modeToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="editorAction"/>
   <addaction name="rulerAction"/>
   <addaction name="moveAction"/>
  </widget>
  <widget class="QToolBar" name="viewToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="zoomInAction"/>
   <addaction name="zoomOutAction"/>
   <addaction name="separator"/>
   <addaction name="trailAction"/>
   <addaction name="vectorsAction"/>
   <addaction name="gridAction"/>
  </widget>
  <action name="stopAction">
   <property name="text">
    <string>Стоп</string>
   </property>
  </action>
  <action name="startAction">
   <property name="text">
    <string>Пуск</string>
   </property>
  </action>
  <action name="addItemAction">
   <property name="text">
    <string>Новый</string>
   </property>
  </action>
  <action name="speedUpAction">
   <property name="text">
    <string>Быстрее</string>
   </property>
  </action>
  <action name="speedUpUpAction">
   <property name="text">
    <string>Еще быстрее</string>
   </property>
  </action>
  <action name="speedDownAction">
   <property name="text">
    <string>Медленее</string>
   </property>
  </action>
  <action name="speedDownDownAction">
   <property name="text">
    <string>Еще медленее</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="loadAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
  <action name="rulerAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Линейка</string>
   </property>
  </action>
  <action name="moveAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Перемещение</string>
   </property>
  </action>
  <action name="editorAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Редактор</string>
   </property>
  </action>
  <action name="trailAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Хвост</string>
   </property>
  </action>
  <action name="vectorsAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Вектора</string>
   </property>
  </action>
  <action name="zoomInAction">
   <property name="text">
    <string>+</string>
   </property>
  </action>
  <action name="zoomOutAction">
   <property name="text">
    <string>-</string>
   </property>
  </action>
  <action name="gridAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Сетка</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
